lib/light-service.rb

```diff
@@ -1,5 +1,6 @@
 require 'logger'
-require 'active_support/core_ext/string'
+require 'i18n'
+require 'structured_warnings'
```

Comment:
If we're requiring `structured_warnings` in `light-service.rb` we probably don't need to re-require them again in the rest of the files, right?

Reply:
Totaly agree: OR in the manifest OR in single files. In both places is just confusing and conceptually wrong.